sklearn/feature_selection/tests/test_base.py

```diff
@@ -1,12 +1,15 @@
 import numpy as np
 from scipy import sparse as sp
+from scipy.stats import pearsonr, spearmanr
```

Comment:
These metrics are not sufficient. You're not providing a case when the scorer can output one statistic (and not provide a p-value).

Maybe you can use the variance function here.